docs/api/tuf.ngclient.rst

```diff
@@ -0,0 +1,17 @@
+ngclient
+========
+
+The ngclient module contains a complete TUF client library implementation.
+
+* :doc:`tuf.ngclient.updater` implements the client update workflow
```

Comment:
at least within one document this should work

```
See `Client workflow`_.

.. _Client workflow: https://theupdateframework.github.io/specification/latest/#detailed-client-workflow
```

Reply:
Awesome, let's do that?

Reply:
I added links to the specification as you suggested. I've put them as references though I don't know if it makes sense since I couldn't figure out a way to cross reference them between docs. At least maybe it's more readable like that.